Sources/Sentry/include/SentryScope+PropagationContext.h

```diff
@@ -0,0 +1,13 @@
+#import "SentryScope.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SentryScope ()
+
+// This is a workaround to make the traceId available in the Swift layer.
+// Can't expose the SentryId directly for some reason.
+@property (nonatomic, readonly) NSString *propagationContextTraceIdString;
```

Comment:
Fine, we plan to make the `Scope.propagationContext` readonly public anyways with https://github.com/getsentry/sentry-cocoa/issues/5594. So please don't tackle making it public in this PR. Just FYI.